public/index.php

```diff
@@ -87,6 +87,29 @@ function url_check($value){
 function h($value){
     return htmlspecialchars($value, ENT_QUOTES);
 }
+
+//ある投稿に何個のいいねが押されているかカウントする関数
+//引数
+//$db : PDOインスタンス
+//$post : 投稿のレコード
+//出力
+//likesテーブルにpost_idが$post['id']であるレコードがいくつあるか
```

Comment:
コメント記述ナイスですが、PHPDocという書き方のフォーマットがあります
```suggestion
/**
* ある投稿に何個のいいねが押されているかカウントする関数
* @param $db PDOインスタンス
* @param $post 投稿のレコード
*
* @return likesテーブルにpost_idが$post['id']であるレコードがいくつあるか
*/
```
みたいな

Reply:
ありがとうございます。PHPDocの書き方見てみます！